### PR TITLE
Node webadmin

### PR DESCRIPTION
**Pull request by fmontoto**

### CODE REVIEW

**Comment by fmontoto** [comment]
Review pls

File: tchsm/node-webadmin/webadmin/nodeadmin.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    docker run -d -v ${NODEADMIN_DIR}/conf:/home/nodeadmin/tchsm-nodeadmin/conf \
+               --name tchsm-nodeadmin --net=tchsm-nodeadmin \
+               -p 0.0.0.0:${EXPOSE_NODE_ROUTER_PORT}:2121 \
+               -p 0.0.0.0:${EXPOSE_NODE_SUB_PORT}:2222 tchsm-nodeadmin
+}
+
+function stop {
+    set -e
+
+    docker rm -f tchsm-nodeadmin
```
Comment (Zeeerooo) [inline tchsm/node-webadmin/webadmin/nodeadmin.sh:43]: In docker exist "stop" and "rm" options. Maybe, can be useful have the two options for the script.

**Review by Zeeerooo — APPROVED** [review]
LGTM

File: tchsm/node-webadmin/webadmin/nodeadmin.sh (hunk 1 above)
Comment (fmontoto) [reply tchsm/node-webadmin/webadmin/nodeadmin.sh:43]: As all the persistent data is mounted as a volume in the host machine I don't care about removing the container and starting a new one each time.

**Comment by raticate** [comment]
Lgtm

**Pull request merged**